probes/hasKnownVulnerabilities/def.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: hasKnownVulnerabilities
+short: Check whether the project has known vulnerabilities
+motivation: >
+  This check determines whether the project has open, unfixed vulnerabilities in its own codebase or its dependencies using the OSV (Open Source Vulnerabilities) service. An open vulnerability is readily exploited by attackers and should be fixed as soon as possible.
+implementation: >
+ The implementation uses the VulnerabilitiesData from the raw results.
+outcome:
+  - The probe returns one negative outcome for each vulnerability in the raw results.
+  - If there are no known vulnerabilities from the raw results, the probe returns one positive outcome.
+remediation:
+  effort: High
+  text:
+    - Fix the vulnerabilities in your own code base. The details of each vulnerability can be found on https://osv.dev.
```

Comment:
Let's create links dynamically with OSV data (link to the vuln). You can use this function https://github.com/ossf/scorecard/blob/main/finding/finding.go#L241 to add add metadata:
```
text:
   - Fix the ${{ metadata.osvid }} by following information from https://osv.dev/${{ metadata.osvid }}.
markdown:
   - Fix the ${{ metadata.osvid }} by following information from [OSV](https://osv.dev/${{ metadata.osvid }}).
```

Reply:
Very nice, thanks!